components/Topic.js

```diff
@@ -1,7 +1,39 @@
 import React from "react";
 
-const Topic = () => {
-	return <div>Topic</div>;
+const Topic = ({ topic, backgroundColor, textColor }) => {
+	return (
+		<div>
+			<div
+				style={{
+					justifySelf: "center",
+					width: "max-content",
+					height: "max-content",
+					borderRadius: "1.0rem",
+					borderWidth: "2px",
+					borderColor: "black",
+					paddingTop: "0.75rem",
+					paddingBottom: "0.75rem",
+					// paddingRight: "0.25rem",
+					// paddingLeft: "0.25rem",
+					backgroundColor: backgroundColor,
+					color: textColor,
+					boxShadow: "8px -6px black",
+				}}
+			>
+				<div
+					style={{
```

Comment:
Convert to Tailwind CSS

Reply:
wait I thought I did change everything to tailwind css. Could you explain what you mean here?